.eslintrc.js

```diff
@@ -40,4 +40,12 @@ module.exports = {
       { components: ["Link"], specialLink: ["hrefLeft", "hrefRight"], aspects: ["invalidHref", "preferButton"] },
     ],
   },
+  overrides: [
+    {
+      files: ["src/pages/*.tsx"],
```

Comment:
ありがとうございます！

こちら再帰的になっていますか？
もしや下記なのかなーと🤔

孫ファイルにも適用されるかお試しいただけると助かります！😊✨

```suggestion
      files: ["src/pages/**/*.tsx"],
```